src/core/public/ui_settings/ui_settings_api.test.ts

```diff
@@ -142,10 +143,16 @@ describe('#batchSet', () => {
     fetchMock.once('*', {
       body: { settings: {} },
     });
-    fetchMock.once('*', {
-      status: 400,
-      body: 'invalid',
-    });
+    fetchMock.once(
+      '*',
+      {
+        status: 400,
+        body: 'invalid',
+      },
+      {
+        overwriteRoutes: false,
```

Comment:
When declaring multiple mocks with identical matchers it is now required to specify whether they should overwrite or extend.